config/sidekiq_scheduler.yml

```diff
@@ -267,17 +267,17 @@ VBADocuments::FlipperStatusAlert:
   description: "Checks status of Flipper features expected to be enabled and alerts to Slack if any are not enabled"
 
 VBADocuments::ReportMonthlySubmissions:
-  cron: "3 2 1 * *  America/New_York"
+  cron: "0 2 1 * * America/New_York"
```

Comment:
Didn't think the 3 needed to be there, so I updated it to be 2:00 AM ET on the first of the month.